core/src/test/java/com/klaytn/caver/feature/KIP17Test.java

```diff
@@ -0,0 +1,1070 @@
+
+        try {
+            BigInteger total = ownerHandler.totalSupply().send();
+            BigInteger[] tokenIdArr = new BigInteger[]{total, total.add(BigInteger.ONE), total.add(BigInteger.valueOf(2))};
+
+            KlayTransactionReceipt.TransactionReceipt mintReceipt = ownerHandler.mint(mDeployerTxManager.getDefaultAddress(), tokenIdArr[0]).send();
+            checkTxStatus(mintReceipt);
+            List<KIP17.TransferEventResponse> transferEventResponses = ownerHandler.getTransferEvents(mintReceipt);
+            checkTransferEventValue(transferEventResponses.get(0), sZeroAddr, ownerAddress, tokenIdArr[0]);
+
+            mintReceipt = ownerHandler.mint(mDeployerTxManager.getDefaultAddress(), tokenIdArr[1]).send();
+            checkTxStatus(mintReceipt);
+            transferEventResponses = ownerHandler.getTransferEvents(mintReceipt);
+            checkTransferEventValue(transferEventResponses.get(0), sZeroAddr, ownerAddress, tokenIdArr[1]);
+
+            mintReceipt = ownerHandler.mint(mDeployerTxManager.getDefaultAddress(), tokenIdArr[2]).send();
+            checkTxStatus(mintReceipt);
+            transferEventResponses = ownerHandler.getTransferEvents(mintReceipt);
+            checkTransferEventValue(transferEventResponses.get(0), sZeroAddr, ownerAddress, tokenIdArr[2]);
+
+            total = ownerHandler.totalSupply().send();
+
+            //Test
+            BigInteger findIndex = total.subtract(BigInteger.ONE);
+            BigInteger tokenId = ownerHandler.tokenByIndex(findIndex).send();
+            assertEquals(tokenId, tokenIdArr[2]);
+
+            findIndex = findIndex.subtract(BigInteger.ONE);
+            tokenId = ownerHandler.tokenByIndex(findIndex).send();
+            assertEquals(tokenId, tokenIdArr[1]);
+
+            findIndex = findIndex.subtract(BigInteger.ONE);
+            tokenId = ownerHandler.tokenByIndex(findIndex).send();
+            assertEquals(tokenId, tokenIdArr[0]);
+
+        } catch (Exception e) {
+            e.printStackTrace();
+            fail();
+        }
+    }
+
+    //KCT-047
+    @Test
+    public void transferFrom() {
+        KIP17 ownerHandler = KIP17.load(mContractAddress, mCaver, mDeployerTxManager, new DefaultGasProvider());
+        String ownerAddress = mDeployerTxManager.getDefaultAddress();
+        String userAddress = mTesterTxManger.getDefaultAddress();
+
+        try {
+            BigInteger tokenId = BigInteger.valueOf(2000);
+
+            KlayTransactionReceipt.TransactionReceipt receipt = ownerHandler.mint(ownerAddress, tokenId).send();
+            checkTxStatus(receipt);
+
+            List<KIP17.TransferEventResponse> transferEventResponses = ownerHandler.getTransferEvents(receipt);
+            checkTransferEventValue(transferEventResponses.get(0), sZeroAddr, ownerAddress, tokenId);
+
+            KlayTransactionReceipt.TransactionReceipt transactionReceipt = ownerHandler.transferFrom(ownerAddress, userAddress, tokenId).send();
```

Comment:
In the test for '***From' function, it would be better to make sure that the token owner and message sender are not the same.

Reply:
Okay, I added Test code. 
Please see test code KCT-061 ~ KCT-066  :)